can/generated_2021/uwrt_mars_rover_can_dump.txt

```diff
@@ -1,4 +1,4 @@
-================================= Messages =================================
+[?1049h[22;0;0t[1;11r(B[m[4l[?7h[11;1H[?1049l[23;0;0t
[?1l>================================= Messages =================================
```

Comment:
?

Reply:
So for some reason this garbage string gets generated each time we run the CAN autogen script. Manually deleting it every time isn't a good solution, will investigate into this later

Reply:
created #34 for ya